apps/asap-server/src/validation/user.validation.ts

```diff
@@ -0,0 +1,90 @@
+import { UserPatchRequest } from '@asap-hub/model';
+import { JSONSchemaType } from 'ajv';
+import { validateInput } from './';
+
+const userInputValidationSchema: JSONSchemaType<UserPatchRequest> = {
+  type: 'object',
+  properties: {
+    jobTitle: { type: 'string', nullable: true },
+    onboarded: { type: 'boolean', nullable: true },
+    contactEmail: { type: 'string', nullable: true },
+    firstName: { type: 'string', nullable: true },
+    lastName: { type: 'string', nullable: true },
+    degree: {
+      type: 'string',
+      enum: [
+        'BA',
```

Comment:
Same again here if we could just use the model array of these in a spread with the extra options so we can change it one place and have it filter through to where it needs to be